src/classlibnative/bcltype/system.cpp

```diff
@@ -321,7 +321,7 @@ INT32 QCALLTYPE SystemNative::GetProcessorCount()
 {
     QCALL_CONTRACT;
 
-    INT32 processorCount = 0;
+    DWORD processorCount = 0;
```

Comment:
I would rather make just one line change here and add cast to the assignment to this variable.

Reply:
done